code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -1,3 +1,142 @@
+/obj/structure/constructshell/large
+	name = "large empty shell"
+	icon = 'icons/obj/cult_large.dmi'
+	icon_state = "shell_narsie_grey"
+	desc = "An oversized construct shell, fit for an elder god. Only a lunatic would even dream of such a crazed contraption."
+	pixel_x = -16
+	pixel_y = -16
+	density = 1
+	anchored = 0
+	var/maxhealth = 200
+	var/health = 200
+	var/orbs = 0
+	var/orbs_needed = 1
+	var/animate_alpha = 0
+	var/image/black_overlay = null
+	var/time_to_win = 120 //2 minutes

```

Comment:
It's actually less than 12 seconds, which doesn't make any sense(it looks like it should be 12) but it's like 1.2 seconds.